libcxx/modules/std.compat.cppm.in

```diff
@@ -17,38 +17,17 @@ module;
 
 // The headers of Table 24: C++ library headers [tab:headers.cpp]
 // and the headers of Table 25: C++ headers for C library facilities [tab:headers.cpp.c]
-#include <algorithm>
-#include <any>
-#include <array>
-#if !defined(_LIBCPP_HAS_NO_ATOMIC_HEADER)
-#  include <atomic>
-#endif
-#if !defined(_LIBCPP_HAS_NO_THREADS)
-#  include <barrier>
-#endif
-#include <bit>
-#include <bitset>
 #include <cassert>
 #include <cctype>
```

Comment:
Let's try to remove the duplicated headers as much as possible. Maybe we can improve it by something like:

```
export module std.compat;
export import std;

export using double_t = std::double_t;
```

Reply:
I don't want to make this change in this PR and investigate it separately. I want to make sure it doesn't make it harder for libc++ to test our modules. I created a new issue for it.

Reply:
Fair enough.